reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -1,35 +1,56 @@
 package org.ihtsdo.termserver.scripting.reports.release;
 
-import java.io.IOException;
+import java.io.*;
 import java.util.*;
 
 import org.ihtsdo.otf.exception.TermServerScriptException;
 import org.ihtsdo.termserver.scripting.ReportClass;
 import org.ihtsdo.termserver.scripting.reports.TermServerReport;
 import org.snomed.otf.scheduler.domain.*;
 import org.snomed.otf.scheduler.domain.Job.ProductionStatus;
-import org.snomed.otf.script.dao.ReportSheetManager;
 
 /**
  * 
  */
 public class PackageComparisonReport extends TermServerReport implements ReportClass {
-	
-	public static final String A_RELEASE = "A Release";
-	public static final String B_RELEASE = "B Release";
-	
-	//public static final String PREV_DEPENDENCY = "Previous Dependency";
-	//public static final String THIS_DEPENDENCY = "This Dependency";
+
+	public static final String PREVIOUS_RELEASE = "Previous Release";
+	public static final String PREVIOUS_RELEASE_ZIP = "Previous Release Package";
+	public static final String CURRENT_RELEASE = "Current Release";
+	public static final String CURRENT_RELEASE_ZIP = "Current Release Package";
+	public static final String PUBLISHED_FOLDER = "Published Folder";
+
+	private final String scriptName = "run_compare.sh";
+	private String[] scriptArguments;
 	
 	public static void main(String[] args) throws TermServerScriptException, IOException {
 		Map<String, String> params = new HashMap<>();
+		params.put(PREVIOUS_RELEASE, "EE Release 2021-11-30");
+		params.put(PREVIOUS_RELEASE_ZIP, "SnomedCT_ManagedServiceEE_PRODUCTION_EE1000181_20211130T120000Z.zip");
+		params.put(CURRENT_RELEASE, "EE Release 2022-05-30");
+		params.put(CURRENT_RELEASE_ZIP, "SnomedCT_ManagedServiceEE_PRODUCTION_EE1000181_20220530T120000Z.zip");
+		params.put(PUBLISHED_FOLDER, "ee");
 		TermServerReport.run(PackageComparisonReport.class, args, params);
 	}
-	
-	public void init (JobRun run) throws TermServerScriptException {
-		getArchiveManager().setPopulateReleasedFlag(true);
-		ReportSheetManager.targetFolderId = "1od_0-SCbfRz0MY-AYj_C0nEWcsKrg0XA"; //Release Stats
-		super.init(run);
+
+	@Override
+	protected void preInit() throws TermServerScriptException {
+		runHeadless(5);
+	}
+
+	@Override
+	protected void init (JobRun run) throws TermServerScriptException {
+		scriptArguments = new String[]{
+				"\"" + run.getParamValue(PREVIOUS_RELEASE) + "\"",
+				"\"" + run.getParamValue(PREVIOUS_RELEASE_ZIP) + "\"",
+				"\"" + run.getParamValue(CURRENT_RELEASE) + "\"",
+				"\"" + run.getParamValue(CURRENT_RELEASE_ZIP) + "\"",
+				"\"" + run.getParamValue(PUBLISHED_FOLDER) + "\""
+		};
+		//getArchiveManager().setPopulateReleasedFlag(true);
+		//ReportSheetManager.targetFolderId = "1od_0-SCbfRz0MY-AYj_C0nEWcsKrg0XA"; //Release Stats
+		suppressOutput = true;
```

Comment:
What does this suppress?   A code comment would help an old brain remember.

Reply:
@pgwilliams Again, this should not be here, was not the part of my PR, it has been changed since, this is some old code resurrected :-(